service/worker/server.go

```diff
@@ -243,28 +244,33 @@ func Start(ctx context.Context, app dom.AppInfo, conf *Config) error {
 
 	if conf.Api.Enabled {
 		handlers := api.GrpcHandlers(conf.Storage, s3Clients, taskClient, rc, policySvc, versionSvc, storageSvc, locker, rpc.NewProxyClient(appRedis), rpc.NewAgentClient(appRedis), notifications.NewService(s3Clients), &app)
-		start, stop, err := api.NewGrpcServer(conf.Api.GrpcPort, handlers, tp, conf.Log, app)
+		start, stop, err := api.NewGrpcServer(conf.Api.GrpcPort, handlers, tp, conf.Log, conf.Api, app)
 		if err != nil {
 			return err
 		}
 		err = server.Add("grpc_api", start, stop)
 		if err != nil {
 			return err
 		}
-		start, stop, err = api.GRPCGateway(ctx, conf.Api, func(ctx context.Context, mux *runtime.ServeMux, endpoint string, opts []grpc.DialOption) error {
-			err = pb.RegisterChorusHandlerFromEndpoint(ctx, mux, endpoint, opts)
+		if conf.Api.Secure {
+			logger.Info().Msg("http gateway for tls grpc api is currently not supported")
```

Comment:
i am not sure that it is a good idea to disable rest api in this case. is it possible to use the same tls config also for tls?